projects/oid4vc-ts

```diff
@@ -0,0 +1,87 @@
+# Project Name
+
+OID4VC TypeScript
+
+# Preferred Maturity Level
+
+Labs
+
+# Project Description
+
+The OpenID for Verifiable Credentials set of standards provide integration of existing OpenID and OAuth2 standards with verifiable credentials.
+
+OID4VC TypeScript is a generic, and low-level implementation of specifications. Currently it provides APIs for Issuers, Wallets, Clients, Authorization Servers and Resource Providers, covering OpenID for Verifiable Credential Issuance, and OAuth2. In addition, it supports a wide range of other OAuth2 related specifications for interoperability and security (such as Pushed Authorization Requests, Demonstrating Proof of Possesion, and PKCE).
+
+Having experience with integration of these standards into another OWF project, Credo, we were able to create a lower-level reusable library that handles the complex parts of the specification, while still allowing full control over the full flow.
+
+In the coming months this project will be extended with support for OpenID for Verifiable Presentations and related specifications.
+
+The goal of this library is to stay as agnostic as possible to the credential formats being used. This allows to keep the dependencies minimal, while also making it less opiniated. Depdning libraries or frameworks (such as Credo) can provide the needed credential formats, crypto suites, and persistance.
+
+The library was created by Animo as part of the SPRIN-D Funke wallet competition.
+
+# Alignment with the OpenWallet Foundation Mission
+
+The DCQL library provides a reusable library that is crucial to wallet interoperability. It is implementing the standards that are being adopted as part of the EU Digital Identity Wallet, as well as other regions in the world (such as California's drivers license project which also supports OID4VC).
```

Comment:
Suggesting changing to "OID4VC TypeScript".

Reply:
Doh 🤦 thanks a lot. This is what you get when writing three proposals at the same time